adapters/oidc/js/src/main/resources/keycloak.js

```diff
@@ -276,6 +280,7 @@
                 };
 
                 var options = {};
+                options.scope = kc.scope;
```

Comment:
I think this is not necessary. `kc.skope` will be used within the `login` function directly.

Reply:
Yep, this was a left over from my experiments.

Reply:
Just removed this line. Build is running.